news/4.36/platform_isv.md

```diff
@@ -5,7 +5,25 @@
 ## Platform Changes
 -->
 
-<!--
 ---
 ## SWT Changes
--->
+
+### Lazy Handle Generation for Images and Fonts (Windows only)
+
+When creating resources like image `Image` and `Font`, the necessary OS handles used to be created eager upon instantiation of the resource.
+On Windows, this behavior was changed for images and fonts.
+Handles are now generated upon first access to it.
+Until then, only the metadata to generate the handles is stored.
+
+This is particularly relevant on Windows because of the introduced [monitor-specific scaling](platform.md#monitor-specific-ui-scaling-as-default-windows-only).
+That requires all resources to be potentially available at different zooms at the same time, depending on the zooms of the monitors on which application windows are placed.
+The lazy loading is one contribution to mitigate an increasing number of OS handles while reducing unnecessary OS handle generation in general.
+
+More precisely, images now only store their data source (such as a file path, a data provider or the like) upon instantiation.
+In case the image data of an image is retrieved (`getImageData(zoom)`), the data is loaded from the data source for the specified zoom, still without creating an OS handle out of it.
```

Comment:
```suggestion
In case the image data of an image is retrieved (`getImageData(zoom)`), the data is loaded from the data source for the specified zoom, still without creating an OS handle for it.
```